python/cudf_polars/cudf_polars/testing/plugin.py

```diff
@@ -168,7 +168,6 @@ def pytest_configure(config: pytest.Config) -> None:
     "tests/unit/datatypes/test_struct.py::test_struct_agg_all": "Needs nested list[struct] support",
     "tests/unit/constructors/test_structs.py::test_constructor_non_strict_schema_17956": "Needs nested list[struct] support",
     "tests/unit/io/test_delta.py::test_read_delta_arrow_map_type": "Needs nested list[struct] support",
-    "tests/unit/lazyframe/test_collect_schema.py::test_collect_schema_parametric": "https://github.com/pola-rs/polars/issues/23214",
```

Comment:
Hmm, it is surprising to me that this test now passes because the bug on the polars side appears to still be there...

Reply:
This was suggested by https://github.com/rapidsai/cudf/pull/19491#pullrequestreview-3071318802

Reply:
Hmm, let me look